modules/manually-configure-iam-nutanix.adoc

```diff
@@ -8,54 +8,55 @@
 
 Installing the cluster requires that the Cloud Credential Operator (CCO) operate in manual mode. While the installation program configures the CCO for manual mode, you must specify the identity and access management secrets.
 
+[NOTE]
+====
+By default, `ccoctl` creates objects in the directory in which the commands are run. To create the objects in a different directory, use the `--output-dir` flag. This procedure uses `<ccoctl_output_dir>` to refer to this directory.
+====
+
 .Prerequisites
 
 * You have configured the `ccoctl` binary.
-* You have an existing `install-config.yaml` file.
-* You have set the following environment variables:
-+
-`NUTANIX_ENDPOINT`:: Specifies the endpoint for Prism Central.
-`Nutanix_PORT`:: Specifies the port that is used to log into Prism Central.
-`NUTANIX_USER`:: Specifies the username that is used to log into Prism Central.
-`NUTANIX_PASSWORD`:: Specifies the password that is used to log into Prism Central.
+
+* You have an `install-config.yaml` file.
 
 .Procedure
 
-. Edit the `install-config.yaml` configuration file so that the `credentialsMode` parameter is set to `Manual`.
+. Create a YAML file that contains the credentials data in the following format:
 +
-.Example `install-config.yaml` configuration file
+.Credentials data format
 [source,yaml]
 ----
-apiVersion: v1
-baseDomain: cluster1.example.com
-credentialsMode: Manual <1>
-compute:
-- architecture: amd64
-  hyperthreading: Enabled
-...
+credentials:
+- type: basic_auth <1>
+  data:
+    prismCentral: <2>
+      username: <username_for_prism_central>
+      password: <password_for_prism_central>
+    prismElements: <3>
+    - name: <name_of_prism_element>
+      username: <username_for_prism_element>
+      password: <password_for_prism_element>
 ----
-<1> Add this line to set the `credentialsMode` parameter to `Manual`.
+<1> Specifies the authentication type. Only basic authentication is supported.
+<2> Specifies the Prism Central credentials.
+<3> Optional: Specify the Prism Element credentials.
++
+[NOTE]
+====
+The default location for the YAML file is in `<home_directory>/.nutanix/credentials`. To specify a different directory, use the `--credentials-source-filepath` flag. This procedure uses `<path_to_credentials_file>` to refer to this directory.
```

Comment:
It is not super clear here on where to use `--credentials-source-filepath` flag. We should mention somewhere that it needs to be used with `cccotl`

Reply:
May be we should put some reference to step 3?

Reply:
Ah, good idea - I can just add it as an annotation where it is used. Should probably do that for other ccoctl procedures too but will start here :)